docs/Getting Started/Debian/Debian Bullseye Root on ZFS.rst

```diff
@@ -0,0 +1,1117 @@
+.. highlight:: sh
+
+Debian Bullseye Root on ZFS
+===========================
+
+.. contents:: Table of Contents
+  :local:
+
+Overview
+--------
+
+Caution
+~~~~~~~
+
+- This HOWTO uses a whole physical disk.
+- Do not use these instructions for dual-booting.
+- Backup your data. Any existing data will be lost.
+
+System Requirements
+~~~~~~~~~~~~~~~~~~~
+
+- `64-bit Debian GNU/Linux Bullseye Live CD w/ GUI (e.g. gnome iso)
+  <https://cdimage.debian.org/mirror/cdimage/unofficial/non-free/cd-including-firmware/current-live/amd64/iso-hybrid/debian-live-11.0.0-amd64-standard+nonfree.iso>`__
```

Comment:
one problem with `current-live` is it will silently turn into `bookworm` when that (Debian 12) is released... I'm not sure there's a good way around that, because the version-specific link:

<https://cdimage.debian.org/mirror/cdimage/release/11.1.0-live/amd64/iso-hybrid/>

... includes the minor version number, which will change more frequently.

Reply:
You're right, but we didn't have a problem with `current-live` in previous versions, so I propose to leave it.